tests/test_openmc_utils.py

```diff
@@ -262,11 +262,11 @@ def test_get_result_error_from_openmc_sp():
 
 def test_create_meshtally():
     if not HAVE_PYMOAB or sys.version_info[0] == 2:
```

Comment:
I don't think we support python 2 anymore, so maybe we can get rid of that part of the test while we're here...